source/grid/manifold_lib.cc

```diff
@@ -32,10 +32,10 @@ SphericalManifold<dim,spacedim>::SphericalManifold(const Point<spacedim> center)
 
 
 template <int dim, int spacedim>
-Point<spacedim>
+Tensor<1,spacedim>
 SphericalManifold<dim,spacedim>::get_periodicity()
 {
-  Point<spacedim> periodicity;
+  Tensor<1,spacedim> periodicity;
   periodicity[spacedim-1] = 2*numbers::PI; // theta and phi period.
```

Comment:
Yes. The comments refer to 2d and 3d. In 2d it is periodic in theta, while in 3d it is periodic in phi.